.github/actions/buster/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:buster
+
+LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
+
+RUN grep -q "# deb-src" /etc/apt/sources.list && sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list || \
```

Comment:
Can you remove some of this duplication via Dockerfile `ARG`s?

https://blog.dockbit.com/templating-your-dockerfile-like-a-boss-2a84a67d28e9

Reply:
https://github.community/t5/GitHub-Actions/Feature-Request-Build-args-support-in-Docker-container-actions/td-p/37802
docker build with `--build-arg` is not supported yet. I wish it was implemented.

Reply:
This is unfortunate -- please upvote the request.

Can we hack around this with shell scripts or similar?

Reply:
From what I heard from @kumagi before this PR, the workaround would involve something like "docker on docker on docker," which sounded not very reasonable to me.

I'm not a big fan of this approach either; it's natural to ask why we can't use the build arg or whatnot. That's one of the big reasons I said there should be a document: we should document why we take this approach.

Reply:
OK, I'll insert some comments in the script and generated files.